### DISCUSSION TITLE
How are you going to sustain OneJs

### DISCUSSION THREAD

**Discussion by Kay2dan** [onejs/one#123] [General]
Hi, One.js sounds amazing.

I'm looking to migrate my app from an existing React based framework. Unfortunately, this framework is in maintenance mode now & Im wary of moving to another framework so in its infancy, which is why I've been leaning toward Supabase/ Convex. However, OneJs would solve so many of my 'wishlist' features that I cant help but be super excited for you guys!

What are you guys thinking in terms of the future? How is One.js going to be sustained? Also, is One.js solving a particular organisation's needs or came about just as a proof of concept?

**Comment by natew** [comment]
Appreciate the kind words. Tamagui today does make enough to pay salaries for our small team. 

Still, we are exploring a raise, as we think long-term it's not sustainable to compete with well-funded alternatives otherwise.

**Reply by natew** [reply]
As for the last question - absolutely it solves our needs. We've been selling the [Takeout stack](https://tamagui.dev/takeout) for a year now and by far the biggest source of complaints and problems has been around the monorepo, two frameworks, two bundlers, and hellish amount of configuration and glue code that took.

I've been working in this space now for a long time - [here's a demo video](https://www.youtube.com/watch?v=HHTYHm6qLFY) I made for a startup I had 8 years ago, it may feel familiar, if there's one thing I am it is relentless, and I feel that One actually is just the start - there's so much interesting we have here to solve.

**Reply by Kay2dan** [reply]
Cheers! Excited & will download & start playing with One.js!
